simtools/util/general.py

```diff
@@ -647,3 +650,69 @@ def change_dict_keys_case(data_dict, lower_case=True):
         raise
 
     return _return_dict
+
+
+@u.quantity_input(rotation_angle=u.deg)
+def rotate(rotation_angle, x, y):
+    """
+    Rotate x and y by the rotation angle given in rotation_angle, in degrees.
+    The function returns the rotated x and y values in the same unit given.
+
+    Parameters
+    ----------
+    rotation_angle: astropy.units.deg
+        Angle to rotate the array in degrees.
```

Comment:
Would it be complicated to generalize and accept also rad?

Reply:
not so much, I just implemented it, with the test.